src/main/webapp/index.html

```diff
@@ -119,38 +119,38 @@ <h3 class="panel__section-heading u-vertical-center">Calendar</h3>
           <p class="panel__section-subheading">Selected Calendars:
           <div class="panel__content">
             <div class="panel__content-entry">
-              <p class="panel__text-icon u-text-calendar">M:
+              <p class="panel__text-icon u-text-calendar" id="day-0">M:
               <div>
-                <p class="u-block">3h 4m free (working)
-                <p class="u-block">4h 12m free (personal)
+                <p class="u-block" id="work-0">3h 4m free (working)
+                <p class="u-block" id="personal-0">4h 12m free (personal)
               </div>
             </div>
             <div class="panel__content-entry">
-              <p class="panel__text-icon u-text-calendar">T:
+              <p class="panel__text-icon u-text-calendar" id="day-1">T:
               <div>
-                <p class="u-block">3h 4m free (working)
-                <p class="u-block">4h 12m free (personal)
+                <p class="u-block" id="work-1">3h 4m free (working)
+                <p class="u-block" id="personal-1">4h 12m free (personal)
```

Comment:
You can remove these hardcoded values now that your JavaScript function populates them.

Reply:
Good suggestion. `Resolved`